templates/zerver/register.html

```diff
@@ -148,21 +148,23 @@ <h1>{{ _('Create your account') }}</h1>
                 </div>
 
                 {% if require_ldap_password %}
-                <div class="input-box">
-                    <input id="ldap-password" class="required" type="password" name="password" required />
+                <div class="input-box password-div">
+                    <input id="ldap-password" class="required" type="password" name="password" autocomplete="off" required />
                     <label for="ldap-password" class="inline-block">{{ _('Password') }}</label>
+                    <i class="fa fa-eye-slash password_visibility_toggle" role="button" aria-label="{{ _('Toggle password visibility') }}"></i>
                     <span class="help-inline">
                         {{ _('Enter your LDAP/Active Directory password.') }}
                     </span>
                 </div>
                 {% elif password_required %}
-                <div class="input-box">
-                    <input id="id_password" class="required" type="password" name="password"
+                <div class="input-box password-div">
+                    <input id="id_password" class="required" type="password" name="password" autocomplete="off"
```

Comment:
Why isn't this `new-password` autocomplete?  I believe this is the registration form.

Reply:
Yes, should have been new-password. Will update this.